com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home:ro
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
```

Comment:
```suggestion
```

Reply:
I really cannot see a difference

Reply:
the suggestion is to remove them, they are not needed.